backend/Controllers/SightingController.cs

```diff
@@ -26,6 +27,13 @@ public ActionResult<ListResponse<Sighting>> GetSightingsBySpeciesId([FromRoute]
         {
             var sightings = _sightings.GetSightingsBySpeciesId(speciesId);
             return new ListResponse<Sighting>(sightings);
+        }   
+
+        [HttpPost("create")]
```

Comment:
This doesn't need to be at the path `/create` - it's quite fun to think about philosophically, but the information "I am creating a sighting" is actually present in the fact you're sending a POST request, so the path `/create` is redundant!

```suggestion
        [HttpPost]
```

It's a common pattern to have something like `/posts`, where:

* GET to `/posts` = return list of all posts
* POST to `/posts` = create a new post
* PUT to `/posts` = wholesale update an existing post
* GET to `/posts/<id>` = retrieve a specific post
* PATCH to `/posts/<id>` = update part of an existing post
* DELETE to `/posts/<id>` = delete a specific post

No paths like `/posts/<id>/delete` needed! The information about the operation you perform is encoded in the HTTP method.